auto_submit/lib/service/config.dart

```diff
@@ -35,28 +35,28 @@ class Config {
 
   Cache get cache => Cache(cacheProvider).withPrefix('config');
 
-  Future<GithubService> createGithubService() async {
-    final GitHub github = await createGithubClient();
+  Future<GithubService> createGithubService(RepositorySlug slug) async {
+    final GitHub github = await createGithubClient(slug);
     return GithubService(github);
   }
 
-  Future<GitHub> createGithubClient() async {
-    String token = await generateGithubToken();
+  Future<GitHub> createGithubClient(RepositorySlug slug) async {
+    String token = await generateGithubToken(slug);
     return GitHub(auth: Authentication.withToken(token));
   }
 
-  Future<String> generateGithubToken() async {
+  Future<String> generateGithubToken(RepositorySlug slug) async {
     // GitHub's secondary rate limits are run into very frequently when making auth tokens.
     final Uint8List? cacheValue = await cache['githubToken'].get(
```

Comment:
You'll need to add the repo.owner to the cache key

Reply:
Hi Casey, do you mean that we should store the related repo info into Cache, and get the value by calling cache.get()?

Reply:
`cache['githubToken-${slug.owner}']`

Reply:
```suggestion
    final Uint8List? cacheValue = await cache['githubToken-${slug.owner}'].get(
```

Reply:
Gotcha, thanks Casey! I updated the code.